aerofiles/seeyou/reader.py

```diff
@@ -196,3 +226,71 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_task_options(self, fields):
+        if not fields[0] == "Options":
+            return
+
+        task_options = {
+            'NoStart': None,
+            'TaskTime': None,
+            'WpDis': False,
+            'NearDis': None,
+            'NearAlt': None,
+            'MinDis': False,
+            'RandomOrder': False,
+            'MaxPts': None,
+            'BeforePts': None,
+            'AfterPts': None,
+            'Bonus': None
+        }
+
+        for field in fields:
+            if field == "Options":
+                continue
+            elif field.split("=")[0] in ["NoStart", "TaskTime"]:                       # string
+                task_options[field.split("=")[0]] = field.split("=")[1]
+            elif field.split("=")[0] in ["WpDis", "MinDis", "RandomOrder"]:            # boolean
+                task_options[field.split("=")[0]] = field.split("=")[1] == "True"
+            elif field.split("=")[0] in ["MaxPts", "BeforePts", "AfterPts", "Bonus"]:  # int
+                task_options[field.split("=")[0]] = int(field.split("=")[1])
+            elif field.split("=")[0] in ["NearAlt"]:
+                task_options[field.split("=")[0]] = float(field.split("=")[1][0:-1])   # float, take of m
+            elif field.split("=")[0] in ["NearDis"]:
```

Comment:
good point, will be changed in both `decode_task_options()` and `decode_task_obs_zone`.